src/ripple/protocol/AnyPublicKey.h

```diff
@@ -102,9 +102,16 @@ class AnyPublicKey
 
 public:
     AnyPublicKey() = delete;
-    AnyPublicKey (AnyPublicKey const&) = delete;
     AnyPublicKey& operator= (AnyPublicKey const&) = delete;
 
+    AnyPublicKey (AnyPublicKey const& other)
+        : buffer_type(other.buffer_type::member.data(),
+            other.buffer_type::member.size())
+        , AnyPublicKeySlice (buffer_type::member.data(),
+            buffer_type::member.size())
+    {
+    }
```

Comment:
This may be a duplicate comment, but if you provide the copy constructor, you should also provide the copy assignment, or include a comment explaining why you aren't.